docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -312,23 +312,16 @@ $ tsh login --proxy=teleport.example.com --user=alice --auth=local
 $ tctl auth sign --user=security-scanner --format=openssh --out=security-scanner --ttl=10h
 ```
 
-Here is a summary of variables and functions we used in this guide:
-
-<table>
-<tr>
-  <th>Variable or Function</th>
-  <th>Description</th>
-</tr>
-<tr>
-  <td>`user.spec.traits`</td>
-  <td>Access traits of local or SSO user.</td>
-</tr>
-<tr>
-  <td>`contains(list, var)`</td>
-  <td>Checks whether list contains variable</td>
-</tr>
-<tr>
-  <td>`equals(var, var)`</td>
-  <td>Checks whether one variable is equal another</td>
-</tr>
-</table>
+### Filter fields explained
+
+Explanation of the fields used in this guide for the `where` conditions.
+
+| Field                                             | Description                                                    |
+|---------------------------------------------------|----------------------------------------------------------------|
+| `user.spec.traits["<list name>"]`                 | the `list` of traits from a local or SSO user                  |
+| `impersonate_role.metadata.labels["<label key>"]` | the label `value` given label `key` from a role to impersonate |
```

Comment:
```suggestion
| `impersonate_role.metadata.labels["<label key>"]` | The label value given the label `key` from a role to impersonate |
```
Since `value` isn't a keyword used in the role resources above or in the table cell to the left of the explanation, I thought it would make sense to remove the code style.